subsys/bluetooth/host/conn.c

```diff
@@ -52,6 +52,8 @@ struct tx_meta {
 	 * continuations).
 	 */
 	bool is_cont;
+	/* Indicates whether the ISO PDU contains a timestamp */
+	bool iso_has_ts;
```

Comment:
Should guard with `CONFIG_BT_ISO`

Reply:
yeah I did that previously, then had to litter the code with ifdefs and IS_ENABLED because of the stone-age language that we use.

Then I noticed that the struct size doesn't actually change, so I left it always in.
An alternative would be converting both to be bits in a bitfield.

Reply:
> Then I noticed that the struct size doesn't actually change, so I left it always in.

That's a fair argument for now, but if someone later needs to add more, then at some point it will increase in size, and then someone with no knowledge of ISO may have to start guarding all the accesses for ISO. I won't block the PR based on this, but this may end up being some technical debt if we don't do it now. 

> An alternative would be converting both to be bits in a bitfield.

That would be a good approach, as we can more easily expand this without increasing size.